### PR TITLE
Remove Pico's isPlayer code.

### PR DESCRIPTION
**Pull request by AbnormalPoof**
Line 490-511 already handles this just fine

### CODE REVIEW

File: source/Character.hx [hunk 1]
```diff
@@ -233,22 +233,10 @@ class Character extends FlxSprite
 				animation.addByPrefix('idle', "Pico Idle Dance", 24);
 				animation.addByPrefix('singUP', 'pico Up note0', 24, false);
 				animation.addByPrefix('singDOWN', 'Pico Down Note0', 24, false);
-				if (isPlayer)
```
Comment (AbnormalPoof) [inline source/Character.hx:236]: The isPlayer code is unnecessary because there's already code that flips the singLEFT/RIGHT animations

**Comment by ThePlank** [comment]
ok i guesse

**Review by ThePlank — APPROVED** [review]

**Comment by AbnormalPoof** [comment]
Will most likely do the same for tankman when the week 7 sauce is out

**Review by MrClogsworthYT — APPROVED** [review]
noice

**Comment by DotWith** [comment]
> 

Pretty Good!

File: source/Character.hx [hunk 2]
```diff
@@ -233,22 +233,10 @@ class Character extends FlxSprite
...
-				else
-				{
-					// Need to be flipped! REDO THIS LATER!
-					animation.addByPrefix('singLEFT', 'Pico Note Right0', 24, false);
-					animation.addByPrefix('singRIGHT', 'Pico NOTE LEFT0', 24, false);
-					animation.addByPrefix('singRIGHTmiss', 'Pico NOTE LEFT miss', 24, false);
-					animation.addByPrefix('singLEFTmiss', 'Pico Note Right Miss', 24, false);
-				}
-
+			        animation.addByPrefix('singLEFT', 'Pico Note Right0', 24, false);
```
Comment (DotWith) [inline source/Character.hx:236]: Maybe format?

**Review by DotWith — APPROVED** [review]

File: source/Character.hx (hunk 2 above)
Comment (AbnormalPoof) [reply source/Character.hx:236]: wdym

**Comment by AbnormalPoof** [comment]
@DotWith hows that?

**Comment by DotWith** [comment]
> @DotWith hows that?

👍

**Comment by MrClogsworthYT** [comment]
just realized that code is for the future when pico becomes a playable character ig you better close this lol!

**Comment by AbnormalPoof** [comment]
It's old code from week 3, leaving it on flips pico's left/right anims when he is selected as player (left note plays the right anim and vice versa)

**Comment by AbnormalPoof** [comment]
This is due to it interfering with this new code

```haxe
if (isPlayer)
		{
			flipX = !flipX;

			// Doesn't flip for BF, since his are already in the right place???
			if (!curCharacter.startsWith('bf'))
			{
				// var animArray
				var oldRight = animation.getByName('singRIGHT').frames;
				animation.getByName('singRIGHT').frames = animation.getByName('singLEFT').frames;
				animation.getByName('singLEFT').frames = oldRight;

				// IF THEY HAVE MISS ANIMATIONS??
				if (animation.getByName('singRIGHTmiss') != null)
				{
					var oldMiss = animation.getByName('singRIGHTmiss').frames;
					animation.getByName('singRIGHTmiss').frames = animation.getByName('singLEFTmiss').frames;
					animation.getByName('singLEFTmiss').frames = oldMiss;
				}
			}
		}
	}
```

**Pull request closed without merging**